### PR TITLE
Fix flake8 warnings in benchmarks, categories, codegen, combinatorics, crypto, deprecated, diffgeom and discrete

### PR DESCRIPTION
**Pull request by oscarbenjamin**
Fixes flake8 warnings in a bunch of packages.

NO ENTRY

### CODE REVIEW

**Comment by oscarbenjamin** [comment]
This is ready for review

File: sympy/benchmarks/bench_symbench.py [hunk 1]
```diff
@@ -54,13 +54,13 @@ def uniq(x):
 
 def bench_R6():
     "sum(simplify((x+sin(i))/x+(x-sin(i))/x) for i in range(100))"
```
Comment (ethankward) [inline sympy/benchmarks/bench_symbench.py]: Do you know what the point of having these strings here is?

Comment (oscarbenjamin) [reply sympy/benchmarks/bench_symbench.py]: I'm not sure but I guess they show in the output when running the benchmarks.

Maybe sympy/benchmarks should be entirely removed now that we have the sympy_benchmarks repo.

**Comment by oscarbenjamin** [comment]
Thanks!

**Pull request merged**